### PR TITLE
DROID-2829 Splash Screen | Enhancement | Updated splash screen and added animation on app launch

### PR DESCRIPTION
**Pull request by Nadezhda-Gurova**
---

- [ ] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        Box(
+            modifier = Modifier
+                .size(size)
+                .background(brush = gradient, shape = CircleShape)
+                .border(borderWidth, borderColor)
+        )
+    }
+}
+
+@Preview(showBackground = true)
```
Comment (konstantiniiv) [inline app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt:117]: we have a @DefaultPreviews annotation now for Previews

**Review by konstantiniiv — APPROVED** [review]

File: app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt (hunk 1 above)
Comment (Nadezhda-Gurova) [reply app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt]: changed on @DefaultPreviews

File: app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+fun PulsatingCircleScreen() {
+    val context = LocalContext.current
+
+    var initialAnimationFinished by remember { mutableStateOf(false) }
+
+    var circleSize by remember { mutableStateOf(16.dp) }
+
+    LaunchedEffect(Unit) {
+        animate(
+            initialValue = 16f,
```
Comment (uburoiubu) [inline app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt:49]: let's export this floats into constants with some clear naming

File: app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            targetValue = 128f,
```
Comment (uburoiubu) [inline app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt:50]: idem

File: app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+            animationSpec = tween(durationMillis = 1000, easing = FastOutLinearInEasing)
+        ) { value, _ ->
+            circleSize = value.dp
+        }
+        initialAnimationFinished = true
+    }
+
+    val infiniteTransition = rememberInfiniteTransition(label = "")
```
Comment (uburoiubu) [inline app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt:58]: `const val TRANSITION_LABEL = "Pulsating transition" `

**Review by uburoiubu — APPROVED** [review]

**Comment by uburoiubu** [comment]
please attach a video with demonstration

File: app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt (hunk 2 above)
Comment (Nadezhda-Gurova) [reply app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt]: exported floats into constants

File: app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt (hunk 3 above)
Comment (Nadezhda-Gurova) [reply app/src/main/java/com/anytypeio/anytype/ui/splash/PulsatingCircleScreen.kt]: Exported float values into constants

**Pull request merged**